cmd/terramate/cli/cloud.go

```diff
@@ -885,6 +992,82 @@ func (c *cli) setGitlabCIMetadata(md *cloud.DeploymentMetadata) {
 	md.GitlabCICDMergeRequestApproved = mrApproved
 }
 
+func (c *cli) setBitbucketPipelinesMetadata(md *cloud.DeploymentMetadata) {
+	md.BitbucketPipelinesBuildNumber = os.Getenv("BITBUCKET_BUILD_NUMBER")
+	md.BitbucketPipelinesPipelineUUID = os.Getenv("BITBUCKET_PIPELINE_UUID")
+	md.BitbucketPipelinesCommit = os.Getenv("BITBUCKET_COMMIT")
+	md.BitbucketPipelinesWorkspace = os.Getenv("BITBUCKET_WORKSPACE")
+	md.BitbucketPipelinesRepoSlug = os.Getenv("BITBUCKET_REPO_SLUG")
+	md.BitbucketPipelinesRepoUUID = os.Getenv("BITBUCKET_REPO_UUID")
+	md.BitbucketPipelinesRepoFullName = os.Getenv("BITBUCKET_REPO_FULL_NAME")
+	md.BitbucketPipelinesBranch = os.Getenv("BITBUCKET_BRANCH")
+	md.BitbucketPipelinesTag = os.Getenv("BITBUCKET_TAG")
+	md.BitbucketPipelinesParallelStep = os.Getenv("BITBUCKET_PARALLEL_STEP")
+	md.BitbucketPipelinesParallelStepCount = os.Getenv("BITBUCKET_PARALLEL_STEP_COUNT")
+	md.BitbucketPipelinesPRID = os.Getenv("BITBUCKET_PR_ID")
+	md.BitbucketPipelinesDestinationBranch = os.Getenv("BITBUCKET_PR_DESTINATION_BRANCH")
+	md.BitbucketPipelinesStepUUID = os.Getenv("BITBUCKET_STEP_UUID")
+	md.BitbucketPipelinesDeploymentEnvironment = os.Getenv("BITBUCKET_DEPLOYMENT_ENVIRONMENT")
+	md.BitbucketPipelinesDeploymentEnvironmentUUID = os.Getenv("BITBUCKET_DEPLOYMENT_ENVIRONMENT_UUID")
+	md.BitbucketPipelinesProjectKey = os.Getenv("BITBUCKET_PROJECT_KEY")
+	md.BitbucketPipelinesProjectUUID = os.Getenv("BITBUCKET_PROJECT_UUID")
+	md.BitbucketPipelinesStepTriggererUUID = os.Getenv("BITBUCKET_STEP_TRIGGERER_UUID")
+}
+
+func (c *cli) newBitbucketReviewRequest(pr *bitbucket.PR) *cloud.ReviewRequest {
+	createdAt, err := time.Parse(time.RFC3339, pr.CreatedOn)
+	if err != nil {
+		printer.Stderr.WarnWithDetails("failed to parse PR created_on time", err)
+	}
+	updatedAt, err := time.Parse(time.RFC3339, pr.UpdatedOn)
+	if err != nil {
+		printer.Stderr.WarnWithDetails("failed to parse PR updated_on time", err)
+	}
+	var avatarURL string
+	if pr.Author.Links != nil && pr.Author.Links.Avatar != nil {
+		avatarURL = pr.Author.Links.Avatar.Href
+	}
+
+	var changesRequestedCount int
+	var approvedCount int
+	for _, participant := range pr.Participants {
+		state, ok := participant.State.(string)
+		if !ok {
+			continue
+		}
+
+		switch state {
+		case "changes_requested":
+			changesRequestedCount++
+		case "approved":
+			approvedCount++
+		}
+	}
+	rr := &cloud.ReviewRequest{
+		Platform:    "bitbucket",
+		Repository:  c.prj.prettyRepo(),
+		URL:         pr.Links.HTML.Href,
+		Number:      pr.ID,
+		Title:       pr.Title,
+		Description: pr.Summary.Raw,
+		CommitSHA:   c.prj.headCommit(),
+		Draft:       false, // Bitbucket Cloud does not support draft PRs.
+		CreatedAt:   &createdAt,
+		UpdatedAt:   &updatedAt,
+		Status:      pr.State,
+		Author: cloud.Author{
+			Login:     pr.Author.DisplayName,
+			AvatarURL: avatarURL,
```

Comment:
Could you add here the ID as already available?
```suggestion
			AvatarURL: avatarURL,
			ID: pr.Author.UUID,
```

Reply:
Done in #2018